Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/** The master audio manager--loads banks and 
+ contains functions that will start, stop, pause, resume sound events*/
+    
+public class AudioController : MonoBehaviour {
```

Comment:
Extend from Manager<AudioController> (handles singleton implementation of managers).

Reply:
After you've done this, I'd recommend making the functions in this class static so they're more easily accessible. (You can use AudioController.instance if you need to reference the AudioController object in the scene.)